README.md

```diff
@@ -34,6 +34,7 @@ platformName | safaridriver binary can only create WebDriver sessions for deskto
 automationName | Value of automationName must equal to 'Safari' in order to start Safari driver session. Values of automationName are compared case-insensitively.
 browserName | safaridriver can only create WebDriver sessions for Safari. Any value passed to this capability will be changed to 'Safari'.
 browserVersion | safaridriver will only create a session using hosts whose Safari version matches the value of browserVersion. Browser version numbers are prefix-matched. For example, if the value of browserVersion is '12', this will allow hosts with a Safari version of '12.0.1' or '12.1'.
+acceptInsecureCerts | safaridriver will accept navigating to an expired or invalid TLS certificate website. safaridriver 15.4 and above support the capability.
```

Comment:
Makes the browser to ignore the appropriate security warning and thus allows navigation to we sites having invalid or expired TLS certificates. The capability is supported since safaridriver v15.4